core/src/test/java/org/springframework/security/authentication/jaas/JaasGrantedAuthorityTests.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2002-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.authentication.jaas;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.Test;
+import org.springframework.security.authentication.jaas.JaasGrantedAuthority;
+
+/**
+ *
+ * @author Clement Ng
+ *
+ */
+public class JaasGrantedAuthorityTests {
+
+	@Test
+	public void authorityWithNullRoleFailsAssertion() {
+		try {
+			JaasGrantedAuthority authority =
```

Comment:
Thank you for adding these tests. For consistency with other tests in the codebase, would you please change the way you are handling these exceptions to use AssertJ instead? Something similar to:

```java
assertThatThrownBy(() -> ...)
    .isInstanceOf(IllegalArgumentException.class)
```